packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -295,13 +295,25 @@ export default defineComponent({
       await Promise.all(calls.map((p) => p.catch((e) => e)))
     }
 
+    const isFolder = computed(() => {
+      return unref(resource).isFolder
+    })
     const loadPreviewTask = useTask(function* (signal, resource) {
+      if (unref(isFolder)) {
+        return
+      }
       preview.value = yield previewService.loadPreview({
-        space,
+        space: unref(space),
```

Comment:
This line is the actual fix in this PR. The `space` is provided by the sidebar as computed, but we were not accessing the value but the ref itself.

Reply:
Oh boy, you're right, see https://github.com/owncloud/web/blob/admin-settings-pagination/packages/web-app-files/src/components/SideBar/SideBar.vue#L296

We should align that some time in the future...